### PR TITLE
Registration/tc_01_02/incorrect_credentials_existing_email

### PR DESCRIPTION
**Pull request by Elgrueber**

### CODE REVIEW

File: tests/01Registration.spec.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { test } from '../fixtures';
+import { description, tags, severity, epic, step, tms, issue } from 'allure-js-commons';
+import { QASE_LINK, GOOGLE_DOC_LINK, URL_ENDPOINT } from '../testData';
+
+test.describe('Registration', () => {
+    test('TC_02_02 | Verify user is not able to sign up with an existing email)', async ({
```
Comment (annaafanaseva) [inline tests/01Registration.spec.js:6]: shouldn't be 'TC_01_02_01' as in ATC file?

File: tests/01Registration.spec.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        await description('Verify that registration fails when an existing email is used');
+        await issue(`${QASE_LINK}`, 'User Login');
+        await tms(`${GOOGLE_DOC_LINK}i02y4jgrnqlw`, 'ATC_01_02');
+        await epic('Registration');
+
+        await step('Navigate to Home page.', async () => {
+            await page.goto('/');
+        });
+        await homePage.clickSignup();
+        await step('Verify user is on Login page', async () => {
```
Comment (annaafanaseva) [inline tests/01Registration.spec.js:22]: Login -> Sing Up?

File: tests/01Registration.spec.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+test.describe('Registration', () => {
+    test('TC_02_02 | Verify user is not able to sign up with an existing email)', async ({
+        page,
+        homePage,
+        signupPage,
+    }) => {
+        await tags('Registration', 'Negative');
+        await severity('critical');
+        await description('Verify that registration fails when an existing email is used');
+        await issue(`${QASE_LINK}`, 'User Login');
```
Comment (annaafanaseva) [inline tests/01Registration.spec.js:14]: link is missing.. maybe would make sense to replace 'User Login'

File: tests/01Registration.spec.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        await tms(`${GOOGLE_DOC_LINK}i02y4jgrnqlw`, 'ATC_01_02');
```
Comment (annaafanaseva) [inline tests/01Registration.spec.js:15]: same as line 6

Comment (Elgrueber) [reply tests/01Registration.spec.js:15]: Done!

File: tests/01Registration.spec.js (hunk 3 above)
Comment (Elgrueber) [reply tests/01Registration.spec.js:14]: Done!

File: tests/01Registration.spec.js (hunk 1 above)
Comment (Elgrueber) [reply tests/01Registration.spec.js:6]: Done!

**Review by annaafanaseva — APPROVED** [review]

**Pull request merged**